satpy/readers/viirs_sdr.py

```diff
@@ -83,12 +83,44 @@ def _get_invalid_info(granule_data):
                " soub:" + str((granule_data == -999.2).sum()))
     return msg
 
+DATASET_KEYS = {'GDNBO': 'VIIRS-DNB-GEO',
```

Comment:
Are there two different names for the TC and non-TC DNB groups? I forget how they did this. I know that the single GDNBO file has both non-TC and TC geolocation. Older files don't have the TC.

Reply:
Oh, I hadn't noticed DNB had TC now. But why, oh why do they have to do it another way with DNB ? Couldn't they just create a GTCDNB file or something ? :(

Reply:
Haha yep. I had the same reaction when I added it to Polar2Grid how ever many years ago. It kind of makes sense since tools like CLASS and SIPS would have to be updated to support a new file type. Would also likely have to replicate a lot of the DQF and other metadata in GDNBO in this new TC file (and come up with a prefix that doesn't have more letters).

I think of it as learning from their mistakes and taking immediate action...by making the files inconsistent and harder to use.

Reply:
Fixed.